tensorflow/lite/kernels/depthwise_conv_test.cc

```diff
@@ -39,8 +39,73 @@ using ::testing::ElementsAreArray;
 
 class BaseDepthwiseConvolutionOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BaseDepthwiseConvolutionOpModel(
+      TfLiteRegistration* registration, const TensorData& input,
+      const TensorData& filter, const TensorData& output, Padding padding_type,
+      int stride_width, int stride_height,
+      const ActivationFunctionType& fused_activation_function,
+      int dilation_factor = 1) {
+    input_ = AddInput(input);
+    filter_ = AddInput(filter);
+
+    int bias_size = GetShape(filter_)[3];
```

Comment:
could this perhaps be ```const```

Reply:
@shahzadlone , thanks for the review, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit